code/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "private": true,
+  "workspaces": [
+    "workspaces/*"
+  ],
+  "devDependencies": {
+    "@babel/cli": "^7.5.5",
+    "@babel/core": "^7.5.5",
+    "@babel/node": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/register": "^7.5.5",
+    "core-js": "^3.2.1",
+    "eslint-config-airbnb-base": "^13.2.0",
+    "eslint-plugin-import": "^2.18.2",
+    "jest": "^24.8.0",
+    "regenerator-runtime": "^0.13.3"
+  },
+  "dependencies": {
+    "eslint": "^6.1.0"
```

Comment:
Have we decided to use the `^` for packages so we can use the GitHub auto upgrade?

Reply:
Yes, that is the plan. This isn't likely to cause any issue as we are using lock files to prevent the libraries from changing. We were a little too paranoid at the beginning of the project as lockfile had only just been introduced and were unsure that it would prevent packages from being upgraded without our knowledge.